src/components/Tooltip/TooltipRenderedOnPageBody.js

```diff
@@ -81,30 +79,24 @@ class TooltipRenderedOnPageBody extends React.PureComponent {
     }
 
     componentDidUpdate(prevProps) {
-        if (prevProps.text === this.props.text) {
+        if (prevProps.text === this.props.text && prevProps.renderTooltipContent === this.props.renderTooltipContent) {
```

Comment:
```suggestion
        if (prevProps.text === this.props.text) {
```

`prevProps.renderTooltipContent === this.props.renderTooltipContent` will always be `true` making `componentDidUpdate` run every time. We don't want that.